jscomp/runtime/caml_splice_call.ml

```diff
@@ -38,6 +38,21 @@ let spliceApply : obj -> obj -> obj = [%raw{|function(fn,args){
   return fn.apply(null,applied)
 }|}] 
 
+let spliceNewApply : obj -> obj -> obj = [%raw{|function (ctor,args){
```

Comment:
Can't remember what this relies on.
Is the compiler going to rename `ctor` and `args` as required depending on context?

Reply:
It will be emitted as is:

```rescript
let f : (int, int) => int =
  %raw(`function (a, b) { return a + b; }`)

let x = f(1, 2);
```


```js
var f = (function (a, b) { return a + b; });

var x = f(1, 2);
```